app.js

```diff
@@ -27,6 +27,10 @@ app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 // 👇 Start handling routes here
 const index = require('./routes/index');
 app.use('/', index);
+const celebrities = require('./routes/celebrities.routes');
+app.use('/', celebrities);
+const movies = require('./routes/movies.routes');
+app.use('/', movies);
```

Comment:
for shorter syntax you can write it like this:

```
app.use('/', require('./routes/celebrities.routes'));
```

Reply:
thanks! That looks much better